bin/winwrapy.bat

```diff
@@ -12,12 +12,13 @@ REM Windows Server 2003 and later versions. So, for older windows systems (such
 REM as Windows XP) only lines 17 or 20 should be kept in the next code block.
 
 REM returns 0 if python.exe is found
-where /Q python.exe
-IF ERRORLEVEL 1 GOTO else
+REM first try bundled python executable, if it is
+REM not found then use system python
+IF exist "./octpy.exe" GOTO bundledPython
```

Comment:
I'm not sure about `./octpy.exe`: maybe the cwd is somewhere else and this dir is merely in our path?  Have you tested this with `python.exe` installed?